internal/daysteps/daysteps.go

```diff
@@ -1,13 +1,61 @@
 package daysteps
 
+import (
+	"errors"
+	"fmt"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/Yandex-Practicum/tracker/internal/personaldata"
+	"github.com/Yandex-Practicum/tracker/internal/spentenergy"
+)
+
+
 type DaySteps struct {
 	// TODO: добавить поля
+	Steps    int
+	Duration time.Duration
+	personaldata.Personal
 }
 
 func (ds *DaySteps) Parse(datastring string) (err error) {
 	// TODO: реализовать функцию
+
+	dataInput := strings.Split(datastring, ",")
+	
+	if len(dataInput) != 2 {
+		err := errors.New("incorrect data entry")
```

Comment:
нет смысла создавать переменную
```suggestion
		return errors.New("incorrect data entry")
```